src/main/java/de/mprengemann/intellij/plugin/androidicons/model/ImageInformation.java

```diff
@@ -111,9 +114,14 @@ public Format getFormat() {
         return format;
     }
 
+    public Destination getDestination() {
+        return destination;
+    }
+
     public File getTargetFile() {
         return new File(String.format(TARGET_FILE_PATTERN,
                                       exportPath,
+                                      destination.toString().toLowerCase(),
```

Comment:
either specify fixed `Locale` or use an argument for the actual folder name in the `Destination` constructor